javascript/javascript1/week3/CactusI0Interactive.js

```diff
@@ -0,0 +1,66 @@
+const activities =[];
+const emptyArray = [];
+const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
+const today  = new Date();
+const oneActivity = [{ date:today.toLocaleDateString("en-US"), activity: 'Youtube', duration: 30 }]
+function addActivity(activity, duration) {
+    if (isNaN(duration) || duration === "") {
+        console.log("Please put a valid duration")
+    }
+    else {
+        return activities.push({date:today.toLocaleDateString("en-US"), activity:activity, duration:duration})
```

Comment:
you do not need a `return` keyword here. it does no harm but it is not necessary, your funcion would work without that. 
If you decide to leave it your funcion will do its thing and then return a new lenght (after adding the activity) of `anctivities` array.